integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/sort/FieldSearchSortIT.java

```diff
@@ -172,38 +186,68 @@ public void byField() {
 	}
 
 	@Test
-	public void byField_use_alreadyExists_missingValue() {
```

Comment:
Not sure why you are replacing a test that you just added? Shouldn't you squash the two commits, if the first one ends up being useless?